### PR TITLE
Add the default value of task-upload-force

### PR DESCRIPTION
**Pull request by Tiakon**
### Changelog category (leave one):
- Documentation (changelog entry is not required)
 
The original document does not specify the type of task-upload-force directly, which is easy to cause ambiguity.
Shows that the type of task-upload-force is Boolean.

[Basis for modification](https://github.com/ClickHouse/ClickHouse/blob/1cf0952d2868b73238702524ac8679c4ed1a747d/programs/copier/ClusterCopierApp.cpp#L194)

### CODE REVIEW

File: docs/en/operations/utilities/clickhouse-copier.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Parameters:
 - `config` — The path to the `keeper.xml` file with the parameters for the connection to ClickHouse Keeper.
 - `task-path` — The path to the ClickHouse Keeper node. This node is used for syncing `clickhouse-copier` processes and storing tasks. Tasks are stored in `$task-path/description`.
 - `task-file` — Optional path to file with task configuration for initial upload to ClickHouse Keeper.
-- `task-upload-force` — Force upload `task-file` even if node already exists.
+- `task-upload-force` — Force upload `task-file` even if node already exists(default is false).
```
Comment (antaljanosbenjamin) [inline docs/en/operations/utilities/clickhouse-copier.md:46]:
```suggestion
- `task-upload-force` — Force upload `task-file` even if node already exists. Default is false.
```

File: programs/copier/ClusterCopierApp.cpp [hunk 2]
```diff
@@ -89,7 +89,7 @@ void ClusterCopierApp::defineOptions(Poco::Util::OptionSet & options)
                           .argument("task-path").binding("task-path"));
     options.addOption(Poco::Util::Option("task-file", "", "path to task file for uploading in ZooKeeper to task-path")
                           .argument("task-file").binding("task-file"));
-    options.addOption(Poco::Util::Option("task-upload-force", "", "Force upload task-file even node already exists")
+    options.addOption(Poco::Util::Option("task-upload-force", "", "Force upload task-file even node already exists (default is false).")
```
Comment (antaljanosbenjamin) [inline programs/copier/ClusterCopierApp.cpp:92]:
```suggestion
    options.addOption(Poco::Util::Option("task-upload-force", "", "Force upload task-file even node already exists. Default is false.")
```

**Review by antaljanosbenjamin — COMMENTED** [review]
I think the information is valuable, it doesn't have to be between parenthesis, it can be a separate sentence.

**Comment by robot-clickhouse** [comment]
This is an automated comment for commit 8a2ace297541e8c838674b980e37a0f79ad13fe4 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/54099/8a2ace297541e8c838674b980e37a0f79ad13fe4/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Comment by Tiakon** [comment]
image

@antaljanosbenjamin  Hi János, the tag check failed twice. I need to add testable tags. Can you help me add them?

**Comment by antaljanosbenjamin** [comment]
I added the can be tested label. If the tests succeed, then the PR will be merged automatically. Otherwise I can merge it manually.

**Comment by antaljanosbenjamin** [comment]
`test_schema_inference_cache` is broken in #53692

**Pull request merged**